### PR TITLE
Calibrestack

### PR DESCRIPTION
**Pull request by flying-sausages**
Added definitions and icons for `calibre-cs` and `calibre-web`

### CODE REVIEW

File: core/profiles.py [hunk 1]
```diff
@@ -26,6 +26,22 @@ class couchpotato_meta:
     pretty_name = "CouchPotato"
     baseurl = "/couchpotato"
     systemd = "couchpotato"
+    
+class calibrecs_meta:
+    name = "calibre-cs"
+    Pretty_name "Calibre Content Serever"
```
Comment (bakerboy448) [inline core/profiles.py]: typo

Comment (flying-sausages) [reply core/profiles.py]:
```suggestion
    pretty_name = "Calibre Content Server"
```
Gotta double check this

File: core/profiles.py [hunk 2]
```diff
@@ -26,6 +26,22 @@ class couchpotato_meta:
...
+    baseurl= "calibre-cs"
+    systemd = "calibre-cs"
+    multiuser = True
+    check_theD = True
+    
+class calibrweb_meta:
+    name = "calibre-web"
+    Pretty_name "Calibre-Web"
```
Comment (flying-sausages) [inline core/profiles.py]:
```suggestion
    pretty_name = "Calibre-Web"
```

File: core/profiles.py [hunk 3]
```diff
@@ -26,6 +26,22 @@ class couchpotato_meta:
...
+    baseurl= "calibre-web"
```
Comment (flying-sausages) [inline core/profiles.py]:
```suggestion
    baseurl = "/calibre-web"
```

File: core/profiles.py [hunk 4]
```diff
@@ -26,6 +26,22 @@ class couchpotato_meta:
     pretty_name = "CouchPotato"
     baseurl = "/couchpotato"
     systemd = "couchpotato"
+    
+class calibrecs_meta:
+    name = "calibre-cs"
+    Pretty_name "Calibre Content Serever"
+    baseurl= "calibre-cs"
```
Comment (flying-sausages) [inline core/profiles.py]:
```suggestion
    baseurl = "/calibre-cs"
```

**Review by flying-sausages — COMMENTED** [review]
ugh

File: core/profiles.py [hunk 5]
```diff
@@ -26,6 +26,22 @@ class couchpotato_meta:
...
+    
+class calibrecs_meta:
+    name = "calibre-cs"
+    pretty_name = "Calibre Content Server"
+    baseurl = "/calibre-cs"
+    systemd = "calibre-cs"
+    multiuser = True
+    check_theD = True
+    
+class calibrweb_meta:
```
Comment (bakerboy448) [inline core/profiles.py]: typo?

Comment (flying-sausages) [reply core/profiles.py]:
```suggestion
class calibreweb_meta:
```

**Pull request merged**